web-server/v0.4/src/pages/ComparisonSelect/index.js

```diff
@@ -133,7 +135,13 @@ class ComparisonSelect extends React.Component {
               };
               return (
                 <div key={iteration.resultName} style={{ marginTop: 32 }}>
-                  <h2>{iteration.resultName}</h2>
+                  <div style={{ display: 'flex' }}>
+                    <h1>{iteration.resultName}</h1>
+                    <span>
+                      <Tag color="blue">{iteration.controllerName}</Tag>
```

Comment:
We need slight margin on each Tag (`marginLeft: 8` should work) for better readability.